db/seeds.rb

```diff
@@ -1,9 +1,11 @@
-language = Language.where(name: 'Malayalam', short_code: 'ml', default: true).first_or_create!
+language = Language.where(name: "Malayalam", short_code: "ml", default: true).first_or_create!
```

Comment:
Can we set this rule in robocop if we prefer double quote over single quote?

Reply:
Shouldnt it be the other way? Only string that require evaluation should be in double quote, right?

Reply:
Yes, that is the default in rubocop. But that is opiniated, for instance rails prefers double quote by default.

Reply:
> Can we set this rule in robocop if we prefer double quote over single quote?
 
@redtachyons sure, let's do that in #39.